docs/getting-started/create-host.md

```diff
@@ -158,11 +159,11 @@ const renderItem: RenderItemFunction = (item, index, level) => {
         return null;
     }
 
-    const { label, linkProps, additionalProps } = item;
+    const { label, to, linkProps, additionalProps } = item;
 
     return (
         <li key={`${level}-${index}`}>
-            <Link {...linkProps} {...additionalProps}>
+            <Link to={isDynamicTo(to) ? to() : to} {...linkProps} {...additionalProps}>
```

Comment:
Is there a simpler way we could do this, with an util method like `resolveTo` which takes a to and return a string

Reply:
I am not sure it would be that useful because the general use case is that you would provide argument to the dynamic to function